wolfcrypt/src/sp_c64.c

```diff
@@ -2843,9 +2843,9 @@ int sp_RsaPublic_2048(const byte* in, word32 inLen, mp_int* em, mp_int* mm,
 {
 #ifdef WOLFSSL_SP_SMALL
     sp_digit* d = NULL;
-    sp_digit* a;
-    sp_digit* m;
-    sp_digit* r;
+    sp_digit* a = NULL;
```

Comment:
These changes will need to be made in the scripts repo and when made you will also see the same ones in c32. You can reproduce with `CFLAGS"-DSP_WORD_SIZE=32"`